lib/cmd/fh3/credential/create.js

```diff
@@ -0,0 +1,105 @@
+/* globals i18n */
+var fhreq = require('../../../utils/request.js');
+var fs = require('fs');
+
+module.exports = {
+  'desc' : i18n._('Create a credential bundle'),
+  'examples' : [{
+    cmd : 'fhc credential create --bundleName=<bundle-name> --bundleType=<bundle-type> --platform=<platform> --env=<env> --privateKey=<private-key> --certificateAndroid=<certificate-android>',
+    desc : 'Create a credential for Android',
```

Comment:
Tks for the review. 👍 I agree 100%.  'credential bundle' instead of just 'credential' is more clear. :-) Done.